gemini/use-cases/retrieval-augmented-generation/rag_qna_with_bq_and_featurestore.ipynb

```diff
@@ -0,0 +1,1004 @@
6x50/H6eGG0u3q7oW+f1+430vBeTqDfWh42vPanOCeiQu88w9+Fnyf1zk5g30UxO+VYH+8RwABBJIS2MdNPP3r7MZJHYV+EUAAAQQQQAABBBBAAAEEEEAAAQQQ2IsENITnB2vmmYK5r7as98KmehVqeuFdZ1eJpjn2fsumoUuXbl7lDftZ34UbtVxolF3b268pu/PPz+dbd263d1fPdQHfKi/01DCzzSrX9ebLbF2tUcZq4PwcM7/7qvr1A1dBq+dv5db1Vr6kC48r1jDNRavQKzdVqsFz0XPw/ppPbMV3a12/X9vOn3ZZffdsKyTT/Kr5ebbz8lxq2NVpaxd44auuc6sL1TXMrr5vnWq3cs92teDp79H3izatsOnu3NZ8v8m2ufNqVKmONa1S1/TM5Oe8dG8/cyHqLhe87+8CTvWbkyZfzV2q+7ba/VR0z7GeiSbuWT6sVuvUsNE56augtinOv1cKypB+EEAgOQFCzuRs6RkBBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBIQYE7OBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhMg5EzOlp4RQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQCABAULOBFDpEgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEkhP4/6Nbl/ObsaAKAAAAAElFTkSuQmCC)"
```

Comment:
It's not recommended to embed images directly in notebooks, it can make them slow to load.

Please attach the image file to this PR and it can be added to a GCS bucket

Reply:
Changed to use gs://github-repo/ thanks!